test/e2e/tests_definitions.go

```diff
@@ -41,6 +41,14 @@ var (
 				vars: []string{
 					"disable_kubeapi_loadbalancer=true",
 					"subnets_cidr=27",
+					// Use smaller instances in Ireland (the cheapest EU region)
```

Comment:
looks like varsFile would be a better fit for most cases.

Reply:
@kron4eg Updated, PTAL.